bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,18 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	# Note: $ED contains trailing slash
+	if [[ -d "${ED}"etc/init.d ]] ; then
+		shopt -s nullglob
+		for x in "${ED}"etc/init.d/* ; do
+			if [[ $(head -n1 "${x}") =~ ^#!/sbin/runscript$ ]] ; then
+				eqawarn "${x#${ED%/}}: #!/sbin/runscript is deprecated, use #!/sbin/openrc-run instead"
```

Comment:
For consistency,  should probably change this message to have a "QA Notice:" prefix. We could format it like the others, where there's a banner followed by a list of files.